tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -20,7 +20,7 @@ class Camera(Base):
     _secure_camera_frame_locator = (By.CSS_SELECTOR, ".secureAppWindow.active[data-manifest-name='Camera'] iframe")
 
     # Controls View
-    _controls_locator = (By.CSS_SELECTOR, '.controls')
+    _controls_locator = (By.CSS_SELECTOR, '.controls.enabled')
```

Comment:
Why do we change this locator? You don't use it in `wait_for_loading_spinner_hidden`, `wait_for_loading_spinner_hidden` or `tap_capture`.